parsers/dataField/parseSingleNFTTransfer.go

```diff
@@ -11,12 +11,12 @@ func (odp *operationDataFieldParser) parseSingleESDTNFTTransfer(args [][]byte, f
 		return responseParse
 	}
 
-	if core.IsSmartContractAddress(parsedESDTTransfers.RcvAddr) {
-		responseParse.Function = parsedESDTTransfers.CallFunction
+	if len(parsedESDTTransfers.ESDTTransfers) == 0 || !isASCIIString(string(parsedESDTTransfers.ESDTTransfers[0].ESDTTokenName)) {
```

Comment:
I think order here for these checks should be the same as in `parsers/dataField/parseMultiESDTNFTTransfer.go`

Reply:
done.